cmd/containerd/builtins_windows.go

```diff
@@ -24,4 +24,5 @@ import (
 	_ "github.com/containerd/containerd/runtime/v2"
 	_ "github.com/containerd/containerd/snapshots/lcow"
 	_ "github.com/containerd/containerd/snapshots/wcow"
+	_ "github.com/containerd/containerd/snapshots/windows"
```

Comment:
Why is this needed? It's referenced from `github.com/containerd/containerd/snapshots/wcow`, so should already be pulled in?

Reply:
Oh weird. When I was experimenting, the files for "github.com/containerd/containerd/snapshots/windows" weren't getting pulled in which caused build issues in "github.com/containerd/containerd/snapshots/wcow".

Reply:
Interesting. Generally this `import _` pattern is only needed when we want to get the side effects from an `init` routine but nothing actually references the package anywhere.

Reply:
Removed.